schemas/json/layout/layout.schema.v1.json

```diff
@@ -964,6 +1059,66 @@
           "$ref": "expression.schema.v1.json#/definitions/boolean"
         }
       }
+    },
+    "autocomplete": {
+      "type": "string",
+      "default": "off",
+      "enum": [
```

Comment:
Would be nice to have some descriptive text here, preferably with a link to somewhere you can read more about the different choices! :blush: When working in vscode with no other context, `"autocomplete": "impp"` makes little sense.

Also, is the default really `off`? :thinking: In other words, do we block browser autocompletion by default?

Reply:
I looked into the URL pointed to in #27, but it had no listing of on/off, at least.. Would be nice to have a source clarifying what those mean.

Reply:
Have updated the issue with a link that also includes the `on` and `off` option.

Default could be `on` if that is what we want. @Febakke might help us answer what the default for this value should be.

Reply:
Correct me if I'm wrong here, but my cursory reading suggests that:
- `off` as the default instructs the browser to never suggest to autocomplete fields. Might be a bit harsh to turn that off for everyone overnight.
- `on` as the default encourages the browser to remember this field forever and _please, please_ suggest previously filled values for the user.

I haven't extensively tested it, so I may very well be wrong here, but I'm leaning against `on` as the default, or possibly just not define it by default (if that's an option).

Reply:
* the `impp` option is removed,
* the default is set top `on`

Reply:
What was it before we added this? Was the default `off` from before? 

The WCAG issue is only to add the attribute if field is one of the values from the list. So on `on` and `off` is not part of this issue. So best to keep it as is if possible.

Reply:
Allright, the WCAG issue decides it then - no attribute by default (seems like the safest option for backwards compatibility as well).

Reply:
Ok, removed "required" and default values.